### ISSUE TITLE
Add currencyReceived and currencySent to SwapResult rpc message

### ISSUE THREAD

**Issue by sangaman** [ExchangeUnion/xud#670]
Currently the `SwapResult` grpc message includes `amountReceived` and `amountSent` but no indication of what currency was received and which was sent. Previously it would have been possible to determine this based on `pairId` and whether the quantity was negative (sell) or positive (buy), but since we changed how that worked to make `quantity` always positive, it's not possible without looking up the order. I suggest adding `currencyReceived` and `currencySent` to make it clear exactly which currencies changed hands.

**Comment by kilrau** [comment]
This looks relatively small - this or next milestone?

**Comment by sangaman** [comment]
Yeah it's small, but I think lets leave it for next milestone because I want to implement it after #669 is merged.
